### DISCUSSION TITLE
Custom book HTML template possible?

### DISCUSSION THREAD

**Discussion by jtbayly** [quarto-dev/quarto-cli#3488] [Q&A]
Currently there seems to be only one book template with multiple "themes" (actually just color schemes, I think). 

I've searched the documentation and the discussions, and I can't seem to find if it's possible to create a new HTML template. I feel like I've even asked this before and been pointed to the bootstrap variables that I can change. 

I really wonder if it's possible to make my own template though.

**Comment by jimjam-slam** [comment]
I think it's worth noting that you can do a lot with the theming: as well as overriding Bootstrap variables, you can [use all of the other powers](https://quarto.org/docs/output-formats/html-themes-more.html) of SCSS or CSS to change styles.

That said, if there are structural changes you feel you can't accomplish with the theming, you could look at [HTML template partials](https://quarto.org/docs/journals/templates.html#html-partials). They're in the journal format docs, but I _assume_ they work for other formats like books? Okay, I actually don't know. But hopefully that's a mechanism by which you can pop the hood a bit more!

**Reply by dragonstyle** [reply]
This is a good suggestion - the template partials will allow you to address the structure of the pages within the book, but the navigation (e.g. navbar, sidebar) and site 'envelope' are not provided by templates, so need to be controlled via website/book options, themes, or scss variables...
